spec/spec_helper.rb

```
@@ -0,0 +1,22 @@
+ENV['RACK_ENV'] = 'test'
+require 'simplecov'
+require 'rack/test'
+require 'rspec'
+
+# Start SimpleCov and configure it
+SimpleCov.start do
+  enable_coverage :branch  # Enable branch coverage for more detailed insights
+  add_filter "/spec/"       # Exclude the 'spec' directory from coverage results
+end
+
+require File.expand_path '../../app.rb', __FILE__  # Adjust to the path of your main app file
```

rubocop

[convention] Use `expand_path('../app.rb', __dir__)` instead of `expand_path('../../app.rb', __FILE__)`.

(Style/ExpandPathArguments)